packages/components/src/components/popover/popover-styles.ts

```diff
@@ -209,12 +209,23 @@ export const getComponentCss = (direction: PopoverDirection, theme: Theme): stri
       transform: 'translate3d(0,0,0)', // Fixes movement on hover in Safari
     },
     spacer: {
-      position: 'absolute',
+      ...(isNative
+        ? {
+            position: 'absolute',
+            overflow: 'initial',
```

Comment:
Isn't this the default?

Reply:
Yes but needed to overwrite the native popover styling